Bottom: f99c8b590caf284a2a1e87cea04bf592794bc0b3
Top:    d5d6f8c16c2a3dbfbd4e125f0917c0b53e9037fd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-08 13:12:28 +0000

TEMPORARY: Comment out some DrawMark calls which might be out of context

Should fix this properly with notificatons to the GUI


---

diff --git a/src/action.c b/src/action.c
index e03026f..91eb8be 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4021,7 +4021,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark (true);
+//	  DrawMark (true);
 	  Marked.status = false;
 	}
       else
@@ -4029,16 +4029,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark (false);
+//	  DrawMark (false);
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark (true);
+//      DrawMark (true);
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark (false);
+//      DrawMark (false);
     }
   return 0;
 }
